scenarioo-server/src/main/java/org/scenarioo/rest/application/ScenariooRestApplication.java

```diff
@@ -0,0 +1,64 @@
+package org.scenarioo.rest.application;
+
+import org.scenarioo.rest.base.AbstractBuildContentResource;
+import org.scenarioo.rest.builds.BranchBuildsResource;
+import org.scenarioo.rest.builds.BuildsImporterResource;
+import org.scenarioo.rest.configuration.BranchAliasesResource;
+import org.scenarioo.rest.configuration.ConfigurationResource;
+import org.scenarioo.rest.configuration.LabelConfigurationsResource;
+import org.scenarioo.rest.diffViewer.*;
+import org.scenarioo.rest.objectRepository.CustomTabsResource;
+import org.scenarioo.rest.objectRepository.GenericObjectsResource;
+import org.scenarioo.rest.objectRepository.ObjectStepResource;
+import org.scenarioo.rest.scenario.ScenariosResource;
+import org.scenarioo.rest.search.SearchResource;
+import org.scenarioo.rest.sketcher.issue.IssueResource;
+import org.scenarioo.rest.sketcher.scenarioSketch.ScenarioSketchResource;
+import org.scenarioo.rest.sketcher.stepSketch.SketchImageResource;
+import org.scenarioo.rest.sketcher.stepSketch.StepSketchResource;
+import org.scenarioo.rest.step.ScreenshotResource;
+import org.scenarioo.rest.step.StepResource;
+import org.scenarioo.rest.usecase.UseCasesResource;
+
+import javax.ws.rs.core.Application;
+import java.util.HashSet;
+import java.util.Set;
+
+public class ScenariooRestApplication extends Application {
```

Comment:
I propose to merge this class with the existing class "ScenariooWebApplication" - this might help to ensure that the rest services dont get created before other parts of our server are configured and instantiated correctly - which is currently a potential problem I occasionally run into when referencing some of our other singleton internal services from REST services on instantiation (which is too early currently).